client/layout/masterbar/woo.scss

```diff
@@ -156,7 +156,7 @@ $breakpoint-mobile: 660px;
 	a:not(.social-buttons__button),
 	.wp-login__links a,
 	.wp-login__links button,
-	.button:not(.social-buttons__button),
+	.button,
```

Comment:
These don't exist anymore, all social buttons were migrated to core buttons in https://github.com/Automattic/wp-calypso/pull/100807